packages/profile-sync-controller/src/controllers/authentication/AuthenticationController.test.ts

```diff
@@ -583,6 +617,12 @@ function createMockAuthenticationMessenger() {
   mockCall.mockImplementation((...args) => {
     const [actionType, params] = args;
     if (actionType === 'SnapController:handleRequest') {
+      if (typeof params === 'string') {
```

Comment:
Why was this added?

Reply:
I think TypeScript is not able to determine what `params` even though _we_ know what `actionType` is. Maybe we also have better types than we did before, somehow. Either way it makes sense, we should have had this all along.